test/db/SQLJSPackageDB.test.ts

```diff
@@ -29,6 +29,15 @@ describe('SQLJSPackageDB', () => {
       expect(packageDb).toBeDefined();
       expect(packageDb.isInitialized()).toBe(true);
     });
+
+    it('should create an initialize a new SQLJSPackageDB using a provided SQL', async () => {
```

Comment:
```suggestion
    it('should create and initialize a new SQLJSPackageDB using a provided SQL', async () => {
```

Reply:
Fixed in [73ee511](https://github.com/FHIR/fhir-package-loader/pull/52/commits/73ee5112c4370cb04b68c431590a0a4f99cd83fb) because I changed the test a bit too. I also asked a question about this test below.